### DISCUSSION TITLE
how to open tooltip programmatically?

### DISCUSSION THREAD

**Discussion by lorenzocampanile** [Spiderpig86/Cirrus#171] [Q&A] [answered]
There is any way I can trigger the tooltip open with JS instead of mouse over? Thank you :-)

You made a great CSS library!

**Comment by Spiderpig86** [comment, marked as answer]
Hi! This is not something currently supported and I can track this for a future release. https://github.com/Spiderpig86/Cirrus/projects/10

**Comment by MacFJA** [comment]
If the element is _focusable_ you can display the tooltip by activating the focus on the element.
But note that the tooltip will disappear as soon as the focus is lost
